File: app/src/main/java/com/concough/android/models/PurchasedModel.java
package com.concough.android.models;

import java.util.Date;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by abolfazl on 7/16/17.
 */

public class PurchasedModel extends RealmObject {
    @PrimaryKey
    public int id = 0;
    public String username = "";
    public int downloadTimes = 0;
    public Boolean isDownloaded = false;
    public Boolean isImageDownloaded = false;
    public Boolean isLocalDBCreated = false;
    public String productType = "Entrance";
    public String productUniqueId = "";
    public Date created = new Date();
}

File: app/src/main/java/com/concough/android/models/PurchasedModel.kt
package com.concough.android.models

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.util.*

/**
 * Created by abolfazl on 7/12/17.
 */

class PurchasedModel: RealmObject() {
    @PrimaryKey
    var id: Int = 0
    var username: String = ""
    var downloadTimes: Int = 0
    var isDownloaded: Boolean = false
    var isImageDownloaded: Boolean = false
    var isLocalDBCreated: Boolean = false
    var productType: String = "Entrance"
    var productUniqueId: String = ""
    var created: Date = Date()
}